Library/JacobianVectorProduct/Mat_Free_Jac.m

```diff
@@ -26,7 +26,8 @@
 function Av = Mat_Free_Jac(t,y,v, rhs, fcn0, normy)
 
 myeps = 10^-7 * max(10^-5 , normy);

```

Comment:
I would like to see a justification of this, and perhaps a user-configurable option?